internal/initialize/command.go

```diff
@@ -15,28 +15,21 @@ import (
 	"github.com/brevdev/brev-go-cli/internal/terminal"
 )
 
-func NewCmdInit(t *terminal.Terminal) *cobra.Command {
-	var project string
+func NewCmdClone(t *terminal.Terminal) *cobra.Command {
+	var name string
 
 	cmd := &cobra.Command{
-		Use:         "init",
+		Use:         "clone",
+		Short:       "Clone a Brev Project",
 		Annotations: map[string]string{"project": ""},
-		Short:       "Initialize a Brev Project",
-		Long:        "Use this to initialize a Brev project.",
-		Example: `  // To init new project in current directory
-  brev init
-
-  // To init existing project
-  brev init <project_name>`,
+		Long:        "Clone an existing Brev project",
+		Example: ` // To clone your existing Brev project
+		brev clone --name your_project_name`,
```

Comment:
```suggestion
		Example: `  // To clone your existing Brev project
  brev clone --name your_project_name`,
```

This is annoying, but within the tick marks go with preserve the string exactly as you provide. So the helptext here looks like:

```
Initialize a Brev project.

Usage:
  brev init [flags]

Examples:
 // To init new project in current directory
  			brev init
```

With the above change you'll get:

```
Clone an existing Brev project

Usage:
  brev clone [flags]

Examples:
  // To clone your existing Brev project
  brev clone --name your_project_name
```